docs/user_data_deletion_requests.md

```diff
@@ -0,0 +1,62 @@
+---
+title: (Coming Soon) User Data Deletion Requests API
+---
+
+:::info
+NOTE: This is not quite rolled out yet! We are just sharing this as a preview of what it will look like. If you have feedback or feature requests, please let us know.
+:::
+
+GDPR and similar laws may require that you delete a user's data when they request it. To support deletion requests on our side, we have built this API for you to call.
+
+### How to use the API
+
+All requests must include the STATSIG-API-KEY field in the header. The value should be a Console API Key which can be created in the Project Settings on console.statsig.com/api_keys.
+
+## Sending Requests
+
+Data deletion requests take the following parameters:
+- `unit_type`: The unit type that corresponds to the IDs that you would like to deleted. This could be userID, stableID, or some custom ID that you have set up
+- `ids`: A list of strings, that are the IDs you would like to delete data for
```

Comment:
Do we want to update this to comma deliminated?

Reply:
updated